_data/backup.yml

```
@@ -157,6 +157,15 @@ websites:
       - totp
     doc: https://www.qnap.com/en-us/how-to/tutorial/article/how-to-enhance-account-security-using-2-step-verification
 
+  - name: STRATO
+    url: https://www.strato.de/
+    img: strato.svg
+    tfa:
+      - totp
+    doc: https://www.strato.de/faq/cloud-speicher/2-Faktor-Authentifizierung/
+    regions:
+      - de
```

```suggestion
      - de
      - nl
```
Also is available for Netherlands: https://www.strato.nl/faq/cloud-storage/Twee-factor-authenticatie/